db/seeds.rb

```diff
@@ -14,30 +14,26 @@
 
 puts "Seeding users..."
 
-user_ids = [] # needed for randomisation of cats and bookings later
 10.times do
   user = User.create!(
     email: Faker::Internet.email,
     password: "secret",
     username: Faker::Movies::LordOfTheRings.character.downcase.split.first
     )
-  user_ids << user.id
 end
 
 puts "Users created."
 puts "Seeding cats..."
 
-cats = [] # needed for randomisation of bookings later
 30.times do
   cat = Cat.create!(
     name: Faker::Name.first_name,
     breed: Faker::Creature::Cat.breed,
     color: Faker::Color.color_name,
     # may be shortened later
     description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat...",
-    user_id: user_ids.sample
+    user_id: User.all.sample.id
```

Comment:
could you please have there `Cat.new` instead of `Cat.create!`, and then instead of assigning user_id assign the 'whole' user by `cat.user = User.all.sample` and then `cat.save!`?